chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -256,20 +252,24 @@
   realm = nil
 end
 
-# Remove rainbows configuration, dating from before the switch to puma
-file "/opt/dell/crowbar_framework/rainbows.cfg" do
-  action :delete
+# Remove rainbows & puma config files, dating from before the switch to passenger
+[
+  "/opt/dell/crowbar_framework/rainbows.cfg",
+  "/etc/sysconfig/crowbar",
+  "/etc/bluepill/crowbar-webserver.pill"
+].each do |old_config|
+  file old_config do
+    action :delete
+  end
 end
 
-template "/etc/sysconfig/crowbar" do
-  source "sysconfig.crowbar.erb"
-  owner "root"
-  group "root"
-  mode "0644"
-  variables(
-    web_host: "127.0.0.1",
-    web_port: web_port,
-  )
+bash "stop old puma process" do
+  code <<-EOF
+    /usr/bin/pumactl -S /opt/dell/crowbar_framework/tmp/pids/puma.state stop
+    rm -f /opt/dell/crowbar_framework/tmp/pids/puma.state
+  EOF
+  ignore_failure true
+  only_if { File.exists?("/opt/dell/crowbar_framework/tmp/pids/puma.state") }
```

Comment:
Lint/DeprecatedClassMethods: `File.exists?` is deprecated in favor of `File.exist?`.